src/components/common/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,39 @@
+import './navbar.css';
+import MainLogo from '../../../assets/icons/logo.svg';
+import Button from '../Button/Button';
+
+const Navbar = ({ user }) => {
+  const handleLoginButton = () => {
+    /** 후에 dom 연결 */
+  };
```

Comment:
핸들러 함수 네이밍은 주로` handle + 이벤트 대상 + 이벤트 이름` 이런식으로 짓는 것 같아요!
```suggestion
  const handleLoginButtonClick = () => {
    /** 후에 dom 연결 */
  };
```